src/writeData/utils/parsePluginsConfigurationText.mjs

```diff
@@ -339,6 +356,26 @@ const formatConfigurationText = configurationText => {
   return configurationText + '\n'
 }
 
+const formatReadmeText = readmeText => {
+  // Change all relative links to Github links
+  const relativeLink = /\(\/plugins\//gi
+  const replacement =
+    '(https://github.com/influxdata/telegraf/tree/master/plugins/'
+
+  return readmeText
+    .map(line => {
+      return line.replace(relativeLink, replacement)
+    })
+    .join()
+}
+
+let newPluginsCount = 0
+
+console.warn(
+  '\x1b[36m%s\x1b[0m',
+  '................................................................................\n'
+)
```

Comment:
what do these characters render?

Reply:
The first argument to `console.warn` is the color code. This is a blue line of dots.